atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfig.java

```diff
@@ -237,7 +244,20 @@ public String type() {
 
     @Override
     public int concurrentGetRangesThreadPoolSize() {
-        return config.concurrentGetRangesThreadPoolSize();
+        // get from install config (to allow overriding in install when runtime is left as default),
+        // otherwise get from runtime config
+        if (config.concurrentGetRangesThreadPoolSize() > 0) {
```

Comment:
This relies on the fact that `concurrentGetRangesThreadPoolSize` will be 0 if `servers` is empty. We should probably be checking the same condition in `servers` and here.

Reply:
Although maybe this is okay?

Reply:
0 is the "default" value. granted it's computed from other defaults, but it's still the default thing I'm comparing to to see if it's an override

Reply:
in principle we should be defensive to overrides that _are_ zero, but that isn't really meaningful for this anyway so I think I'm ok with this.